codegen/src/main/scala/akka/grpc/gen/scaladsl/Service.scala

```diff
@@ -38,7 +38,7 @@ object Service {
     Service(
       fileDesc.scalaPackageName,
       serviceClassName,
-      fileDesc.getPackage + "." + serviceDescriptor.getName,
+      (if (fileDesc.getPackage.isEmpty) "" else fileDesc.getPackage + ".") + serviceDescriptor.getName,
```

Comment:
Maybe we should consider adding a new `FileDescriptorOps` for the logic in one versus two, duplicated.

Reply:
To provide one convention.